CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [5.0.1] - 2017-05-09
```

Comment:
I completely removed the version and left only the date, is this the way you wanted it?